src/detections/filters/context.js

```diff
@@ -0,0 +1,12 @@
+import React, { useContext as useReactContext, useState } from 'react'
+
+export const Context = React.createContext({})
+
+export const useContext = () => useReactContext(Context)
+
+export const useDefaultContext = () => {
+  const [filters, setFilters] = useState([])
+  const [selectItems, setSelectItems] = useState(['Class', 'Score', 'Timestamp', 'Cameras'])
```

Comment:
Let's rename this to something like `filterVariants`

Reply:
done